trending_data/src/main/webapp/unique_window_handler.js

```diff
@@ -163,10 +192,10 @@ export class UniqueWindowHandler {
   }
 
   /**
-  * Loads the current Youtube data
+  * Loads the current Youtube data for given category id
+  * @param {string} categoryId
   */
-  async loadYTData() {
-    this.ytDataDiv = await prepareYTPosts(this.marker);
-    return this.ytDataDiv;
+  async loadYTData(categoryId) {
+    this.ytDataDiv = await prepareYTPosts(this.marker, categoryId);
```

Comment:
Why was the return statement removed?

Reply:
The extra variable `ytContent` used inside `openWindow()` (which stored the return of this method) seemed unnecessary because the data is loaded in `this.ytDataDiv` which can be accessed without being returned. It was also inconsistent with Twitter method which did not return anything.